File: src/classes/api/req_res/get-kmemo-response.ts
// ˅
'use strict';

import { Kmemo } from '@/classes/datas/kmemo';
import { GkillAPIResponse } from '../gkill-api-response';

// ˄

export class GetKmemoResponse extends GkillAPIResponse {
    // ˅
    
    // ˄

    kmemo_histories: Array<Kmemo>;

    constructor() {
        // ˅
        super()
        this.kmemo_histories = new Array<Kmemo>()
        // ˄
    }

    // ˅
    
    // ˄
}

// ˅

// ˄

File: src/server/gkill/api/req_res/get_kmemo_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type GetKmemoResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	KmemoHistories []reps.Kmemo `json:"kmemo_histories"`
}
